opus/import/obs_instrument_ebrocc.py

```diff
@@ -111,38 +120,51 @@ def field_obs_wavelength_wavelength2(self):
     ################################
 
     def field_obs_profile_occ_type(self):
-        return 'STE'
+        return self._create_mult('STE')
 
     def field_obs_profile_occ_dir(self):
         occ_dir = self._index_col('OCCULTATION_DIRECTION')
         if occ_dir in ('INGRESS', 'EGRESS', 'BOTH'):
-            return occ_dir[0]
+            return self._create_mult(occ_dir[0])
         self._log_nonrepeating_error(f'Unknown OCCULTATION_DIRECTION "{occ_dir}"')
-        return None
+        return self._create_mult(None)
 
     def field_obs_profile_body_occ_flag(self):
-        return self._supp_index_col('PLANETARY_OCCULTATION_FLAG')
+        body_occ_flag = self._supp_index_col('PLANETARY_OCCULTATION_FLAG')
+        return self._create_mult(body_occ_flag)
 
     def field_obs_profile_quality_score(self):
-        return ("UNASSIGNED", "Unassigned")
+        data_dict = self._create_mult(
```

Comment:
Make more compact. Also is this necessary. Doesn't "UNASSIGNED" automatically display as "Unassigned"? Original code might be unnecessary.